CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## v1.0.0 (02/12/2019)
+
+**olp-sdk-dataservice-read**
+
+* Added the `OlpClientSetting` class with the `KeyValueCache` instance.
+* Added the `VersionedLayerClient` class that is used to access versioned layers on OLP. This class implements the `getData` and `getPartitions` methods.
+* Added the `VolatileLayerClient` class that is used to access volatile layers on OLP. This class implements the `getPartitions` and `getData` methods.
+* Added the `ArtifactClient` class that is used to access the artifact service on OLP. This class implements the `getSchemaDetails` and `getSchema` methods.
+* Added the `ConfigClient` class that is used to access the configuration service on OLP. This class implements the `getCatalogs` method.
+* Added the `QueryClient` class that is used to access the query service on OLP. This class implements the `fetchQuadTreeIndex` method.
+* Added a possibility to abort requests using `AbortSignal`.
+* Added a possibility to set a billing tag in each request.
+* Added `QuadKeyUtils` that has a set of utilities and functions intended for work with quadkeys: convert a quadkey to a Morton code, convert a Morton code to a quadkey, and validate a quadkey.
+* Added the `RequestFactory` class that is used for look-up requests. This class implements the static `create` method that returns the built `DatastoreDownloadManager` instance with a correct base URL to service, and the `getBaseUrl` method that caches and returns base URLs to services.
+
+**Breaking Changes**
+
+* The `CatalogClient` constructor changed. Now, pass `OlpClientSettings` by a value. Also, `KeyValueCache` is now a part of `OlpClientSettings`.
+* The `getLayer` method was removed from `CatalogClient`. Now, to initialize a deprecated `CatalogLayer`, use the `VersionedLayerClient` or `VolatileLayerClient` classes.
+* The `getTile` method was removed from `CatalogClient`. Now, to get layer data, use `VersionedLayerClient.getData()` or `VolatileLayerClient.getData()`.
+* The `getAgregatedTile` method was removed from `CatalogClient`. Now, to get partitions metadata and layer data, use `VersionedLayerClient.getPartitions()` and `VersionedLayerClient.getData()`
+or `VolatileLayerClient.getPartitions()` and `VolatileLayerClient.getData()`.
+* The `getSchema` and `getSchemaDetails` methods were removed from `CatalogClient`. Now, to get schemas and schema metadata, use the `ArtifactClient` class.
+* The `HypeDataProvider` class was removed. Now, catalog and layer clients can be initialized without additional classes.
+* The `DatastoreClient` class was removed. Now, `CatalogClient` can be initialized without additional classes.
+
+**olp-sdk-dataservice-api**
+
+* Fixed issues with requests to the blob storages
+
+**olp-sdk-authentication**
+
+* Added posibillity to read credentials from file on the NodeJs.
+
+
+
 ## v0.9.2-beta (28/10/2019)
 
 * Changed prepare process before publishing
```

Comment:
Please adapt below to look similar to above or vice versa.
Add dots at the end of the line, remove extra newlines.
Also v0.9.1-beta uses breaking change differently then above. Which one should be the case now?